cli/src/pcluster/models/imagebuilder.py

```diff
@@ -150,4 +170,23 @@ def __init__(
         self.image = image
         self.build = build
         self.dev_settings = dev_settings
-        self._add_validator(BaseAMIValidator, ami_id=self.build.parent_image)
+        self._set_default()
+        self._add_validator(
```

Comment:
This validator should stay in the root volume given that we're validating a root volume's field.